src/types.ts

```diff
@@ -8,6 +8,7 @@
 export type ApiType =
   | "evm"
   | "tendermint-rpc"
+  | "tendermint-http"
```

Comment:
this change is part of this pr or did you push it by mistake?

Reply:
This is the output from the following command:

https://github.com/zeta-chain/networks/blob/0af35613cd3c0508908dc5f3ba8130db1dac4030/package.json#L14

Ideally, it should've been updated earlier. This is not related to this PR, but I thought it could be merged.